src/index.js

```diff
@@ -53,21 +47,129 @@ const createServer = (options) => {
   return new Server(options)
 }
 
+/**
+ *
+ * Create a interface for tests setup
+ *
+ * @param {TestsInterfaceOptions} createOptions
+ * @returns {TestsInterface}
+ */
+const createTestsInterface = (createOptions = {}) => {
+  // Managed nodes
+  const nodes = []
+
+  const createNode = async (options = {}) => {
+    // Create factory with merged options
+    const ipfsFactory = create(merge(
+      {
+        ipfsOptions: {
+          init: { profiles: ['test'] },
+          preload: { enabled: false }
+
+        }
+      },
+      createOptions,
+      options
+    ))
+
+    // Spawn with merged options
+    const node = await ipfsFactory.spawn()
+
+    // Add `peerId`
+    const id = await node.api.id()
+    node.api.peerId = id
```

Comment:
This is a lot less onerous when getting the ID is a one liner...it would be nice to not do this and would speed up the tests that don't actually use the peer ID.

Reply:
theres a lot of tests that use api.peerId and lots of utils that do just this and api.id() is super simple impact should minimal.
to be fair i think we could do this in js-ipfs after init. 

@achingbrain didn't you find something related to this in the orbit-db codebase ?

Reply:
Makes sense, lets leave it for now, we can always remove later.